src/pages/ProductList/ProductList.scss

```
@@ -0,0 +1,201 @@
+@import '../../styles/baseStyle/colors.scss';
+
+html,
+body {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+
+  display: flex;
+  justify-content: center;
+  // box-sizing: inherit;
+}
+
+.productList {
+  box-sizing: border-box;
+  margin-top: 150px;
+  header {
+    .subHeadBottom {
+      h1 {
+        // background-color: lightcoral;
+        margin-top: 30px;
+      }
+      p {
+      }
+    }
+  }
+
+  main {
+    width: 1140px;
+    margin: 0 auto;
+    // justify-content: center;
+    section.headContainer {
+      position: relative;
+      border-bottom: 1px solid lightgrey;
+      article.layoutWrapper {
+        .container {
+          margin-bottom: 35px;
+          padding-top: 40px;
+          button {
+            height: 55px;
```

```suggestion
    .headContainer {
      position: relative;
      border-bottom: 1px solid lightgrey;
      
      .container {
        margin-bottom: 35px;
        padding-top: 40px;
        
        button {
          height: 55px;
```
- 필요한 셀렉터만 사용해주세요
- 같은 맥락에서 이미 클래스 셀렉터로 충분히 특정 dom을 지칭하고 있는데, 태그 셀렉터 까지 같이 정의해줄 필요 없습니다. 유사한 부분이 많으니 전체적으로 훑어보시고 정리 부탁드립니다
- 새로운 셀렉터가 나올 때는 가독성 차원에서 한줄씩 띄워주세요

---

필요한 셀렉터만 사용하고, 태그 셀렉터 같이 정의하기, 새로운 셀렉터 사용시 한줄씩 띄워주기. 참고해서 코드 작성하도록 하겠습니다! 감사합니다